renderer/src/pages/Welcome.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { getFilAddress, getOnboardingCompleted } from '../lib/station-config'
+import { ReactComponent as StationLogoLight } from '../assets/img/station-logo-light.svg'
+
+const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
+
+const Welcome = (): JSX.Element => {
```

Comment:
I like this component for its simplicity and focus, but wonder if we can find a better name. "Welcome" is the purpose to the user, but to the codebase it serves as something else: It checks whether the user as completed onboarding and set up a fil address, and redirects based on those results. Maybe "Redirector" works, it's boring but explicit. I still think overall this isn't the way to go, since now this component needs to know about the requirements of other components. I'm going to create an alternative PR so we can compare solutions :)

Reply:
Right. 
I think this will depend on the way the app evolves. In the meantime, I renamed it to make it more explicit as you suggested.